docs/source/tutorials/using_text.rst

```diff
@@ -0,0 +1,461 @@
+##########
+Using Text
+##########
+
+There are two different way by which you can render **Text** in videos:
+
+1. Using Pango (:mod:`~.text_mobject`)
+2. Using LaTeX (:mod:`~.tex_mobject`)
+
+If you want to render simple Text, you should use either :class:`~.Text` or 
+:class:`~.MarkupText` or one of it's detrivaties like :class:`~.Paragraph`.
+See :ref:`using-text-objects` for more information.
+
+LaTeX should only be used when you need Mathematical Typesetting. See 
```

Comment:
I dislike "only", if one prefers LaTeX typesetting for regular text, they are also free to use it.

```suggestion
LaTeX should be used when you need mathematical typesetting. See 
```